taxi/views.py

```diff
@@ -1,3 +1,15 @@
 from django.shortcuts import render
+from taxi.models import Car, Driver, Manufacturer
```

Comment:
There should be a blank line between the import of Django modules and the import of local modules according to the checklist. Consider adding a blank line after line 1.